tools/ide_plugins/pysa-vscode/config-language-configuration.json

```diff
@@ -0,0 +1,19 @@
+// JSON language-configuration taken from https://github.com/microsoft/vscode/blob/main/extensions/json/language-configuration.json
```

Comment:
Is there a way to include this file and the other `.json` file, rather than straight up copy-pasting them? Eg. something like what you did in `PysaConfig.tmLanguage` to just include the JSON grammar, rather than copy it in.

Reply:
Looks like this is an issue that's been brought up and is [currently in the backlog](https://github.com/microsoft/vscode/issues/2871) - I don't think there's any official VSCode support for it, but I can try the workarounds mentioned if we really don't want to copy and paste. Since language configuration is used for autocomplete etc. within the IDE, I don't think there's any way to go around VSCode to get it :/